src/components/Layout/Navbar/NavbarDesktop.tsx

```diff
@@ -45,23 +37,17 @@ export default function NavbarDesktop() {
    <nav className={styles.navbar}>
      <UnorderedList
       listStyleType="none"
-      sx={{
-        color: "white",
-        background: "#1d1283",
-        display: "flex",
-        padding: "20px",
-        margin: 0,
-        gap: "20px",
-        alignItems: "center",
-      }}
+      className={styles.navbarMenu}
     >
       <ListItem>
+        <Link href={HOME_URL}>
         <Image
           src="/images/Real-Dev-Squad@1x.svg"
-          width={50}
-          height={50}
+          width={45}
+          height={45}
```

Comment:
Hope the size changes has been discussed, didn't see it mentioned in issue.

Reply:
There is a difference  between status site navbar and member site navbar so to bring the consistency in navbar so i have set the size of logo according to status site